internal/Get-Connection.ps1

```diff
@@ -0,0 +1,115 @@
+﻿Function Get-Connection
```

Comment:
There are several ways to resolve an IP address and hostname, including using .NET objects. We should not should rely on underlying methods to determine what a function name should be. 

Initially, I thought this was going to simply resolve primary IP and Hostname/DNS. If it is used to determine how to connect to a server, then it should be Test. Though this is sounding like two commands, no?